packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { css, spacing, Body, Icon } from '@mongodb-js/compass-components';
+import { connect } from 'react-redux';
+import type { ConnectedProps } from 'react-redux';
+import type { Dispatch } from 'redux';
+
+import PipelineStages from './pipeline-stages';
+import PipelineActions from './pipeline-actions';
+import {
+  getSavedPipelines,
+  savedPipelinesListToggle,
+} from '../../../modules/saved-pipeline';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+});
+
+const pipelineTextAndStagesStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const pipelineStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  marginRight: spacing[5],
+});
+
+const pipelineTextStyles = css({
+  marginRight: spacing[1],
+  marginLeft: spacing[1],
+});
+
+const openSavedPipelinesStyles = css({
+  display: 'flex',
+  marginRight: spacing[1],
+  marginLeft: spacing[1],
+  cursor: 'pointer',
+});
+
+const PipelineHeader: React.FunctionComponent<PipelineHeaderProps> = ({
+  onShowSavedPipelines,
+}) => {
+  return (
+    <div className={containerStyles}>
+      <div className={pipelineTextAndStagesStyles}>
+        <div className={pipelineStyles}>
+          <Body weight="medium" className={pipelineTextStyles}>
+            Pipeline
+          </Body>
+          <div
+            role="button"
+            onClick={() => onShowSavedPipelines()}
+            className={openSavedPipelinesStyles}
+            aria-label="Open saved pipelines"
+            tabIndex={0}
+          >
```

Comment:
This needs to be a `button`, not a `div` with a role. Buttons come with a lot accessibility features, not only click handlers, otherwise you need to reimplement all the accessibility features that come with a `button` element yourself and it's a lot of work